### ISSUE TITLE
Data tracking

### ISSUE THREAD

**Issue by jskiiqur** [dmitriy-ilchenko/QrAndBarcodeScanner#144]
Hello there. Why do you need all this data to be collected?
![Screenshot_20220605-090440_DuckDuckGo](https://user-images.githubusercontent.com/106902579/172040956-45060dbe-8fcf-4c81-8570-534ff0c29e4c.png)
![Screenshot_20220605-090343_DuckDuckGo](https://user-images.githubusercontent.com/106902579/172040966-84b7bb59-7846-4350-8f9f-02a93b169565.png)

**Comment by robsmith11** [comment]
I agree that this is concerning.  A QR scanning app has access to possibly sensitive information and shouldn't be "calling home" at all.

**Comment by ghost** [comment]
What is this tracking? I've deleted the app.

**Comment by brgrvrm** [comment]
The app is really great and functional, but trackers in it are a privacy and security concern. Please remove them.

**Comment by ghost** [comment]
I use Binary Eye now from F-Droid, it's not as good, but it's close enough.

**Comment by woheller69** [comment]
in the F-Droid version tracking seems to be disabled

gradle:

    fDroid {
                buildConfigField "boolean", "ERROR_REPORTS_ENABLED_BY_DEFAULT", "false"
            }

and F-droid yml: https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.example.barcodescanner.yml

    gradle:
          - fDroid
    prebuild: sed -i -e '/io.sentry.android.gradle/d' -e '/^sentry {/,/^}/d' build.gradle

**Comment by woheller69** [comment]
Unfortunaly it still sends requests to sentry.io ...

> in the F-Droid version tracking seems to be disabled
> 
> gradle:
> 
> ```
> fDroid {
>             buildConfigField "boolean", "ERROR_REPORTS_ENABLED_BY_DEFAULT", "false"
>         }
> ```
> 
> and F-droid yml: https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.example.barcodescanner.yml
> 
> ```
> gradle:
>       - fDroid
> prebuild: sed -i -e '/io.sentry.android.gradle/d' -e '/^sentry {/,/^}/d' build.gradle
> ```

**Comment by woheller69** [comment]
please create an AndroidManifest.xml variant for F-Droid flavor
and set

     android:value=""

for Sentry to switch it off
See https://gitlab.com/fdroid/fdroiddata/-/issues/2846

**Comment by VasilisKos** [comment]
Link this fact with the fact that the developer doesn't respond at all to all these reported issues and bugs, and you've got a very suspicious situation...

**Comment by woheller69** [comment]
for F-Droid we fixed it by different means :-)
